src/discord/events/message/registerTeam.ts

```diff
@@ -0,0 +1,214 @@
+import Discord from 'discord.js';
+
+import { Config } from '../../../entities/config';
+import logger from '../../../logger';
+import { colors } from '../../constants';
+import { Team } from '../../../entities/team';
+
+class TeamInfo {
+  idArr: string[] = [];
+
+  name = '';
+
+  description = '';
+
+  table = 0;
+
+  user = '';
+
+  constructor(i: string) {
+    this.user = i;
+  }
+
+  addMembers(mem: string[]): void {
+    this.idArr = mem;
+  }
+
+  addUser(): void {
+    this.idArr.push(this.user);
+  }
+
+  addName(n: string): void {
+    this.name = n;
+  }
+
+  addDescription(d: string): void {
+    this.description = d;
+  }
+
+  addTable(t: number): void {
+    this.table = t;
+  }
+}
+
+let registrationInProgress = false;
+let decrement = false;
+let findName: Team;
+let findTable: Team;
+
+const promptArr = [
+  'Who are you hacking with? You have 5 minutes to send a reply. If you are hacking alone, simply write `none`!\n**Be sure to input their Discord ID in a comma-separated list e.g. 111222333444555,666777888999000,...**',
```

Comment:
What was the intent behind time-boxing their reply? I don't think it really adds value for us and it causes us to have to write more tests (and tests that involve time are **very** unpleasant)

Was there a reason we have to use ID's instead of having the user @mention others? If they can @mention others, we should use that approach because asking them to find ID numbers will add a bunch of complexity and inevitably lead to questions...

```suggestion
  'Who are you hacking with? (Reply by @mentioning each of your team mates here. If you're solo, reply with `none`',
```